tools/gen_addons_table.py

```diff
@@ -63,17 +63,15 @@ def replace_in_readme(readme_path, header, rows_available, rows_unported):
         addons.extend([
             '\n',
             '\n',
-            'Available addons\n',
-            '----------------\n',
+            '## Available addons\n',
             render_markdown_table(header, rows_available),
             '\n'
         ])
     if rows_unported:
         addons.extend([
             '\n',
             '\n',
-            'Unported addons\n',
-            '---------------\n',
+            '## Unported addons\n',
```

Comment:
I'd remove this from this PR. I agree it's better, but this requires synchronization with the bot, so it's better to have that separate so I can focus on deploying that separately.

Reply:
I don't understand what you mean... Isn't this what the bot itself writes? So, any change here would be done automatically by it, right?

The reasoning for this is to match prettier heading styles.

In any case, I guess you know what you're talking about, so let me just add a TODO note here. As prettier is disabled in these files for now and, if enabled, the ignore tags would make it not touch these lines, it shouldn't be a problem in any case.